events/migrations/0003_auto_20180626_1841.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-26 18:41
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('events', '0002_auto_20180622_1817'),
```

Comment:
@ericbroder Was this meant to be included on this OR? Fine if so we don't have to rework just want to make sure.